pkgs/development/python-modules/gpib-ctypes/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, pytestCheckHook
+, pytest-runner
+, linux-gpib
+}:
+
+buildPythonPackage rec {
+  pname = "gpib_ctypes";
```

```suggestion
  pname = "gpib-ctypes";
```
Please use a hyphen instead of an underscore.